magento/magento2
# Stopping old queue consumers during upgrade

### Description (*)
Put poison pill after each 'setup:upgrade' execution.

### Fixed Issues (if relevant)
1. Fixes magento/magento2#23540

### Steps for testing:
1. Check "version" value in "queue_poison_pill" table in Magento instance database.
2. Upgrade Magento instance.
3. Verify "version" value in "queue_poison_pill" table has changed in Magento instance database.
4. Execute queue:consumers:restart command.
5. Verify "version" value in "queue_poison_pill" table has changed in Magento instance database.



### Contribution checklist (*)
